browser/net/brave_referrals_network_delegate_helper_unittest.cc

```diff
@@ -21,7 +21,6 @@ using brave::ResponseCallback;
 
 TEST(BraveReferralsNetworkDelegateHelperTest, ReplaceHeadersForMatchingDomain) {
   const std::array<std::tuple<GURL, std::string>, 3> test_cases = {
```

Comment:
this isn't going to compile. need to change array length

Reply:
Hm, it really shouldn't, but it did compile locally (and the unit tests succeeded).

Reply:
Fixed in latest push. Thanks.